src/smt/optimization_solver.h

```diff
@@ -221,14 +245,28 @@ class OptimizationSolver
       bool needsTimeout = false,
       unsigned long timeout = 0);
 
-  /** The parent SMT engine **/
+  /**
+   * Optimize multiple goals in Box order
+   * @return OPTIMAL if all of the objectives are either OPTIMAL or UNBOUNDED;
```

Comment:
Why not returning UNBOUNDED if one of the objectives is unbounded?

Reply:
Good question, because UNBOUNDED is also considered as OPTIMAL, just when you call getValue it will return something like oo (\infty).